packages/codelabs/src/basics/web-components.md

```diff
@@ -178,10 +178,10 @@ class CoolHeading extends HTMLElement {
 ```
 
 <aside class="notice">
-We're adding the event listener in the constructor here because the `connectedCallback` can be called multiple times during the lifecycle of our element, for example when using drag and drop an element is disconnected from one location and connected again in another. That would cause the event listener to be registered multiple times, a common source of bugs.
+We are adding the event listener in the constructor, because the `connectedCallback` method can be called multiple times during the lifecycle of our element. For example when using drag and drop, an element is disconnected from one location and connected again in another. That would cause the event listener to be registered multiple times, a common source of bugs.
```

Comment:
```suggestion
We're adding the event listener in the constructor, because the `connectedCallback` method can be called multiple times during the lifecycle of our element, for example when using drag and drop, or when an element is disconnected from one location and connected again in another. That would cause the event listener to be registered multiple times, a common source of bugs.
```